eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala

```diff
@@ -73,7 +71,9 @@ object ExplorerApi {
             case Some(uri) =>
               context.pipeToSelf(explorer.getLatestHeaders(uri, currentBlockCount)(context)) {
                 case Success(headers) => WrappedLatestHeaders(replyTo, headers)
-                case Failure(e) => WrappedFailure(e)
+                case Failure(e) =>
+                  context.log.error(s"${explorer.name} failed to get latest headers", e)
+                  WrappedFailure(e)
```

Comment:
This isn't necessary, it's already logged in the handler for `WrappedFailure`.
And you generally should never log inside `pipeToSelf` as the actor context is missing so you won't get MDC fields added.